### ISSUE TITLE
[bitnami/solr] Uploading a custom Solr schema.xml file as part of cluster bringup

### ISSUE THREAD

**Issue by mlanner** [bitnami/charts#19184]
### Name and Version

bitnami/solr

### What is the problem this feature will solve?

I've successfully created a Solr Cloud cluster and added my own core and collection. I'm now trying to load an existing `schema.xml` file/contents to override the 'managed schema' that gets created by default. Or at least that is my understanding of the the schema that gets populated by a standard bring up of a cluster. I'm looking for information regarding how one can upload a new schema, but the Helm chart documentation doesn't provide any clear instructions for how to achieve that. Perhaps one of `args` or `extraEnvVars*` parameters is how I could achieve that, but if so, it's unclear to me how I'd use them. I've also tried to do it via the Solr API, but have not been successful so far.

### What is the feature you are proposing to solve the problem?

Documentation for how to achieve the above.

### What alternatives have you considered?

Using Solr API after the cluster comes up, but I have not been able to make it work.

**Comment by rafariossaa** [comment]
Hi,
  Currently there is not an option for this. 
  Maybe you could try to mount a `schema.yaml` file by using `extraVolumes` and `extraVolumesMount` , or you could upload you schema using `zkCli` to zookeeper.

**Comment by mlanner** [comment]
@rafariossaa Thanks for the input. That could be something I'll try.

Another thing I considered was to take the bitnami/solr container image and layer in our own configs, copying in our server settings and the schema and thereby creating our own derivative image. I did try that late last week, but ran into some errors with permissions. It's possible I can work around those, though. Do you have any thoughts on that approach?

Additionally, I did notice in the container image that there's an env var `SOLR_CORE_CONF_DIR`, which lets you re-configure where Solr reads its configuration from. However, as far as I can tell, that is not plumbed through via the Helm chart. Is that something you think could be done? I'm happy to open a specific feature request for that if you think that'd be helpful. Also, if so, I'm also happy to test and help document that feature for the Helm chart.

**Comment by rafariossaa** [comment]
I have no tried, but I think it could be an approach, mounting a volume and setting that environment variable. 
As you indicated, there are not explicit parameters for this, but I think it could be handle by using `extraEnvVars`, `extraVolumes` and `extraVolumeMounts`.

**Comment by OriMeyuhas** [comment]
Hello,

I'm encountering a similar issue, but in my context, I am not using Solr Cloud. I'm attempting to create a core using the `coreNames` value and need to change the default schemas as well. Additionally, I've tried adding another layer to the Docker image to copy the configurations, but I faced some errors and issues with this approach. Did you find any solution to this problem? Any suggestions would be greatly appreciated.

Thank you!
